RELEASE/scripts/autoscend/auto_restore.ash

```diff
@@ -1084,12 +1086,12 @@ boolean __restore(string resource_type, int goal, int meat_reserve, boolean useF
     }
   }
 
-  skill pick_blood_skill(){
-    int conservative_blood_bond_cost = hp_cost($skill[Blood Bond]) + ((9-hp_regen())*10);
+  skill pick_blood_skill(int final_hp){
     boolean bloodBondAvailable = auto_have_skill($skill[Blood Bond]) &&
       auto_have_familiar($familiar[Mosquito]) && //checks if player can use familiars in this run
```

Comment:
side note: we'll never use this in 100% familiar runs if the 100% familiar is not mosquito

Reply:
```suggestion
      canChangeToFamiliar($familiar[Mosquito]) &&
```

Reply:
ok so, my change fixed it for pokefam but not for 100% familiar. it was not trying to switch TO mosquito. IT was asking "is there a reason to ever use a skill that buffs +familiar" and the answer to it is "yes if we can use familiars in this path, no if we can't use familiars in this path".